package-lock.json

```diff
@@ -0,0 +1,7460 @@
+{
```

Comment:
Can you delete this file? and just have `yarn.lock`.
You can use following commands:
1. `yarn dev` to run a dev build.
2. `yarn install` to install new packages.